core/metacling/src/TCling.cxx

```diff
@@ -2442,7 +2444,7 @@ static int HandleInterpreterException(cling::MetaProcessor* metaProcessor,
    }
    catch (cling::InterpreterException& ex)
    {
-      Error("HandleInterpreterException", "%s\n%s", ex.what(), "Execution of your code was aborted.");
+      Error("HandleInterpreterException", "%s.\n%s", ex.what(), "Execution of your code was aborted.");
```

Comment:
This dot was previously removed in #9537 and the reference file was updated, now leading to a failing test. I've prepared #9587 to remove the dot and fix the test again.